src/database/sqlite.rs

```diff
@@ -41,6 +41,9 @@ static MIGRATIONS: &[&str] = &[
     "INSERT INTO transaction_details SELECT txid, timestamp, received, sent, fee, height FROM transaction_details_old;",
     "DROP TABLE transaction_details_old;",
     "ALTER TABLE utxos ADD COLUMN is_spent;",
+    "DELETE FROM utxos WHERE rowid > (SELECT MIN(rowid) FROM utxos u2 WHERE utxos.txid = u2.txid AND utxos.vout = u2.vout);",
```

Comment:
I discussed a bit with @afilini about this, while theoretically the duplicated lines should contain the same info, we can't be so sure about that...
If the duplicated lines contain different info (for example: same `outpoint`, different `is_spent`), we might end up in an inconsistent state.
Alekos suggested wiping the entire db.
I suggested wiping it only if we find some duplicated outpoints with the different info - if this can be done in a single SQL query (should be doable, but I'm not entirely sure). What do you think?

Reply:
But I'm so proud of my little SQL de-duplicating! haha, anyway, rather than risk an inconsistent state I'll update this to always wipe the database, it's only a cache after all. And I have to agree with @afilini to wipe the data even if there are no dups since either way we will need to warn users that it MIGHT be wiped on this release and they'll need to be prepared to handle it with a sync().